timestream.lua

```diff
@@ -0,0 +1,393 @@
+-- speeds up the calendar, units, or both
+
+--[====[
+
+timestream
+==========
+This only affects fortress mode.
```

Comment:
Could you also add a description of what the script does? It can be short, and the comment on line 1 would be a good place to start.
It would also be good to mention that arguments can be skipped by passing `-1`, since there aren't examples for all of them.

Reply:
Done